lib/zuora/objects/amendment.rb

```diff
@@ -6,20 +6,20 @@ class Amendment < Base
     validates_length_of :name, :maximum => 100
     validates_inclusion_of :auto_renew, :in => [true, false], :allow_nil => true
     validates_length_of :code, :maximum => 50, :allow_nil => true
-    validates_datetime_of :contract_effective_date, :allow_nil => true
-    validates_datetime_of :customer_acceptance_date, :allow_nil => true
-    validates_datetime_of :effective_date, :allow_nil => true
-    validates_datetime_of :service_activation_date, :if => Proc.new { |a| a.status == 'PendingAcceptance' }
+    # validates_datetime_of :contract_effective_date, :allow_nil => true
+    # validates_datetime_of :customer_acceptance_date, :allow_nil => true
+    # validates_datetime_of :effective_date, :allow_nil => true
+    # validates_datetime_of :service_activation_date, :if => Proc.new { |a| a.status == 'PendingAcceptance' }
     validates_length_of :description, :maximum => 500, :allow_nil => true
     validates_numericality_of :initial_term, :if => Proc.new { |a| a.type == 'TermsAndConditions' }
     validates_numericality_of :renewal_term, :if => Proc.new { |a| a.type == 'TermsAndConditions' }
-    validates_date_of :term_start_date, :if => Proc.new { |a| a.type == 'TermsAndConditions' }
+    # validates_date_of :term_start_date, :if => Proc.new { |a| a.type == 'TermsAndConditions' }
```

Comment:
Or why this is commented out?

Reply:
Reverted in this commit [841f53b](https://github.com/Sage/zuora/pull/18/commits/841f53b823eaa50e77a234490d96a9cee4bdc218)